### PR TITLE
[CES] AlarmEvent API Update

### LINKED ISSUE #434: [CES] AlarmEvent API Updated
https://confluence.tsi-dev.otc-service.com/display/DR/RM-3379-CES-API?sortBy=date&sortOrder=descending
linked https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/1896

### PR DESCRIPTION
**Pull request by Aloento**
fix https://github.com/opentelekomcloud/gophertelekomcloud/issues/434

=== RUN   TestEvents
--- PASS: TestEvents (0.31s)
PASS

### CODE REVIEW

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]
Where is the tests?

File: acceptance/openstack/ces/v1/events_test.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/ces/v1/events"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestEvents(t *testing.T) {
+	client, err := clients.NewCesV1Client()
+	th.AssertNoErr(t, err)
+
+	eventsRes, err := events.ListEvents(client, events.ListEventsOpts{
```
Comment (anton-sidelnikov) [inline acceptance/openstack/ces/v1/events_test.go:34]: Fails with: Failure in events_test.go, line 18: unexpected error "Resource not found: [GET https://ces.eu-ch2.sc.otc.t-systems.com/V1.0/b4372bf357a54745b311a8bca742feed/event], error message: {\"error_msg\":\"The API does not exist or has not been published in the environment\",\"error_code\":\"APIGW.0101\",\"request_id\":\"63373c12d550d1661dc069c496f34e52\"}\n"

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

**Review by artem-lifshits — COMMENTED** [review]
Hello @Aloento would be nice to get this module done :)
We have 2 issues on terraform side regarding CES

**Comment by Aloento** [comment]
@artem-lifshits can you give me the issues link? thx.

File: acceptance/openstack/ces/v1/events_test.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import (
+	"testing"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/ces/v1/events"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestEvents(t *testing.T) {
```
Comment (artem-lifshits) [inline acceptance/openstack/ces/v1/events_test.go:12]: `CreateEvents` test missing

**Review by artem-lifshits — CHANGES_REQUESTED** [review]

**Review by artem-lifshits — DISMISSED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by artem-lifshits — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**